lib/appraisal/gemfile.rb

```diff
@@ -1,26 +1,14 @@
-require 'appraisal/dependency_list'
-require 'appraisal/gemspec'
-require 'appraisal/git_source'
-require 'appraisal/path_source'
-require 'appraisal/group'
-require 'appraisal/platform'
+require 'appraisal/bundler_dsl'
 
 module Appraisal
-  # Load bundler Gemfiles and merge dependencies
-  class Gemfile
-    attr_reader :dependencies
-
-    def initialize
-      @sources = []
-      @ruby_version = nil
-      @dependencies = DependencyList.new
-      @gemspec = nil
-      @groups = {}
-      @platforms = {}
-      @git_sources = {}
-      @path_sources = {}
-    end
+  autoload :Gemspec, 'appraisal/gemspec'
+  autoload :GitSource, 'appraisal/git_source'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.